great_expectations/validator/validator.py

```diff
@@ -1679,7 +1680,10 @@ def _initialize_expectations(
                     **expectation_suite_dict, data_context=self._data_context
                 )
             else:
+                # burn this
+                old_expectation_suite = expectation_suite
                 expectation_suite = copy.deepcopy(expectation_suite)
+                expectation_suite._store = old_expectation_suite._store
```

Comment:
I've seen a strange error across a couple PRs, where `ExpectationSuite._store` is `None`. Turns out its coming from this code - for some reason the deepcopy isn't copying over the `_store` attribute. In any case, deep copying a class is very likely not what we want in the first place.

Reply:
added logic to the `deepcopy` dunder instead